app/services/medical_copays/request.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module MedicalCopays
+  ##
+  # An object responsible for making HTTP calls to the VBS service
+  #
+  # @!attribute settings
+  #   @return [Config::Options]
+  # @!attribute host
+  #   @return (see Config::Options#host)
+  # @!attribute service_name
+  #   @return (see Config::Options#service_name)
+  # @!attribute url
+  #   @return (see Config::Options#url)
+  class Request
+    extend Forwardable
+    include Common::Client::Concerns::Monitoring
+
+    STATSD_KEY_PREFIX = 'api.medical_copays.request'
```

Comment:
It looks like you'll need to init this statsd key in the statsd initializer

Reply:
I think if I'm calling `with_monitoring` in the `post` method below I should be able to use what's provided in `lib/common/client/concerns/monitoring.rb` to log the stats? The `total` and `failure` stats is all I'm looking to capture for the MVP. For all custom monitoring I will be setting up the config in the stated initializer 👍